### ISSUE TITLE
Properties of undefined (reading 'lang') custom 404 Astro page

### ISSUE THREAD

**Issue by yudyananda** [opral/inlang-paraglide-js#175]
Hi, i just got an error `Properties of undefined (reading 'lang')` when using `paraglide` object from `Astro.locals` in custom 404 page.

```js
// my layout
---
import { languageTag } from "../paraglide/runtime";
---
<!doctype html>
<html lang={languageTag()} dir={Astro.locals.paraglide.dir}>
    ...
</html>
```

Is there a way that we can do from the paraglide side?

**Comment by samuelstroschein** [comment]
Loris is on vacation this and next week. Response will take approx 9 days!

**Comment by LorisSigrist** [comment]
Hi @yudyananda 

I've tried to reproduce this but it seems to work fine. I added a custom 404 page in opral/monorepo#3040. Both `Astro.locals.paraglide.dir` and `languageTag()` seem to work on the 404 page. 
- `/non-existent` -> English 404
- `/de/non-existnent` -> German 404

I know there are some Astro versions where the middleware wasn't being called correctly in production. Which version are you on?
